onadata/libs/utils/common_tools.py

```diff
@@ -48,11 +48,11 @@ def get_boolean_value(str_var, default=None):
     return str_var if default else False
 
 
-def get_uuid():
+def get_uuid(hex: bool = True):
```

Comment:
Codacy found an issue: [Redefining built-in 'hex'](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5989432)